src/Models/UserInput.cs

```diff
@@ -53,6 +55,7 @@ bool assessSqlServicesSeparately
         // Configuration tab
         public List<string> AzureMigrateSourceAppliances { get; }
         public Workflow WorkflowObj { get; }
+        public bool IsAvsProposal { get; }

```

Comment:
Please Explore ENUM suitability?

Reply:
I followed the same format as bool isExpressWorkflow, wanted to maintain uniformity. Also, it becomes better this way. Please do let me know if I still need to change this.